gslib/utils/text_util.py

```diff
@@ -408,3 +409,23 @@ def get_random_ascii_chars(size, seed=0):
   contents = six.ensure_binary(contents)
   random.seed()  # Reset the seed for any other tests.
   return contents
+
+def get_prefix():
```

Comment:
Nit: For new methods, we should use the `GoogleStyleGuideNamingStyle`, `not_the_python_one` :)

Also, this method name isn't super indicative of what the method does. How do you feel about calling this something like `GetLowStringSortingPrefix`?

Reply:
Right! I'll pascal case it, and `GetLowStringSortingPrefix` sounds good!